.github/workflows/ci.yml

```diff
@@ -22,15 +22,31 @@ jobs:
     strategy:
       matrix:
         target: ["nanos", "nanox", "nanosplus"]
+        package: [include_gif, testmacro, ledger_secure_sdk_sys, ledger_device_sdk]
     steps:
       - name: Clone
         uses: actions/checkout@v4
       - name: Cargo clippy
         uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: --target ${{ matrix.target }}
+          args: -p ${{ matrix.package }} --target ${{ matrix.target }}
   
+  clippy-cargo-ledger:
+    name: Run static analysis for cargo-ledger
+    runs-on: ubuntu-latest
+    container:
+        image: ghcr.io/ledgerhq/ledger-app-builder/ledger-app-dev-tools:latest
```

Comment:
Not sure we need to be in the dev-tools container for cargo-ledger jobs... :thinking:

Reply:
Well, the idea is to be consistent with regards to other CI tests: all are performed within dev-tools container, so I have decided to keep the same test environment. For cargo-ledger, we are using a stable toolchain for cargo-ledger, so there's should not be any diff even if we were using an ubuntu-based only container.